apis/extension/resource.go

```diff
@@ -52,6 +52,11 @@ const (
 	// AnnotationResourceStatus represents resource allocation result.
 	// koord-scheduler patch Pod with the annotation before binding to node.
 	AnnotationResourceStatus = SchedulingDomainPrefix + "/resource-status"
+
+	// AnnotationExtendedResourceSpec specifies the resource requirements of extended resources.
+	// It annotates the requests/limits of extended resources and can be used for CRI hooks which cannot get the
+	// original pod spec.
+	AnnotationExtendedResourceSpec = DomainPrefix + "/extended-resource-spec"
```

Comment:
I'm wondering which domain is more appropriate, because we need to consider the scope of this protocol, for example, should the scheduler support this extended protocol? From what I have seen, it may be more appropriate to put it on node domain, only as an internal API.

Reply:
fixed